### ISSUE TITLE
Strange rendering of "Current streak"

### ISSUE THREAD

**Issue by haxpor** [Naramsim/GithubOriginalStreak#59]
Before it works perfectly well, but now at this very moment it's not.

I'm still not sure what is the root cause of the problem. My current streak should be at 50 at this particular moment.

![captured screenshot](http://i.imgur.com/cQ4yO2v.png)

Or you just see my Github [profile](https://github.com/haxpor).

**Comment by Naramsim** [comment]
Hi, I really couldn't replicate it. Which browser are you using?

![streak](https://user-images.githubusercontent.com/8996268/27859001-a0a18e22-6177-11e7-8f9c-ec388d722dbd.PNG)

Moreover, is it happening every time, or just sometimes?

**Comment by haxpor** [comment]
Thank you! It has happened as always since I reported this issue. Still now it also happens.

I tried to clear cache specific to Github domain. Then I see correct data but without logging in to my Github account yet. After I logged in, the problem kicks in again.

I use Chrome (`Version 59.0.3071.115 (Official Build) (64-bit)`) on mac OS High Sierra 10.13.

**Comment by Naramsim** [comment]
Does the error occur only on your profile or on every profile you visit? Do you use other extensions that modify Github? Pressing F12 and looking in the console after a reload shows some messages?

Can you do me a favor? Can you allow this extension to run in incognito, then open an incognito window, log in and see if the error persists?

Thanks for your time.

**Comment by haxpor** [comment]
I answer your questions as below

* Occur only to me
* I didn't use any other Github related extension
* Do you mean developer console on chrome? I didn't see any thing by the way, all clean.
* Still persist even enabled it in incognito mode, I have the same streak number for longest streak, and current streak.

**Comment by Naramsim** [comment]
Hi, i will look into it, hope to find something

**Comment by haxpor** [comment]
Thanks! Sure, just let me know if you need me to test anything.

**Comment by Naramsim** [comment]
Hey @haxpor, I don´t understand a thing. Actually, the picture you've posted to the very top of the issue is presenting the truth. I will post the same down here

![img](https://camo.githubusercontent.com/428d367dcaf0f3de4c7b0c8c68e725a234f255be/687474703a2f2f692e696d6775722e636f6d2f635134794f32762e706e67)

There are exactly 138 squares painted in green (3 +2 +19 * 7 = 138). In your picture, there is no missing green square on May 11, 2017. Maybe it is a private contribution that other users can't see. 

So, I don't see where the issue is 😄

**Comment by haxpor** [comment]
Ohh man, thanks for your time. I think you're right about 11 May, once I used your extension and it renders ~50, this might due to a hole on that date in which it's weird I actually see a hole in incognito mode recently, refresh then it's gone, no missing hole anymore. So might be something about Github itself. I checked on that day, I have 1 contribution and repo is public, so there's no lose of streak.

And you're absolutely right, without that 11 May thing, I count and it's exactly the correct streak data rendering with your extension.

**In short**: at the moment it's all good, and correctly rendered. Thank you for your time!
